packages/frontend/src/components/custom-paper/CustomPaper.tsx

```diff
@@ -1,19 +1,72 @@
 import React from "react";
-import {Box, Paper} from "@mui/material";
+import {Avatar, Box, Paper} from "@mui/material";
+
+type Size = 'small' | 'large';
+
+type CustomPaperStyle = {
+    paper: {
+        width: string
+    }
+    banner: {
+        height: string
+    },
+    avatar: {
+        width: string,
+        height: string,
+        bottom: string,
+        marginBottom: string
+    }
+}
+
+const customPaperStyleMap = new Map<Size, CustomPaperStyle>([
+    ["small",
+        {
+            paper: {
+                width: "30%"
+            },
+            banner: {
+                height: "120px"
```

Comment:
if possible use as much % in css rather then px.

Reply:
In this scenario, I need pixels to be more precise with the avatar but I agree we should do % for a responsive website.
It will take some time to rearrange the CSS to be responsive because it's not just the banner but also the text and inputs aren't so responsive.
Because we have a whole component for the `CustomPaper` we can always go back and fix it later with no problems. In my opinion, we should do it later and focus now on the Inbox, Library, and Home page for the next 2 weeks.
What do you think?